src/main/resources/swagger/api-docs.yaml

```diff
@@ -0,0 +1,165 @@
+swagger: '2.0'
+info:
+  title: GPAlloc
+  description: |
+    Google project allocation service
+  version: '0.1'
+basePath: /
+produces:
+  - application/json
+tags:
+  - name: test
+    description: Test API
+  - name: project
+    description: Projects API
+
+security:
+  - googleoauth:
+      - openid
+      - email
+      - profile
+
+securityDefinitions:
+  googleoauth:
+    type: oauth2
+    authorizationUrl: 'https://accounts.google.com/o/oauth2/auth'
+    flow: implicit
+    scopes:
+      openid: open id authorization
+      email: email authorization
+      profile: profile authorization
+
+##########################################################################################
+## PATHS
+##########################################################################################
+paths:
+
+  '/ping':
+    get:
+      summary: Ping
+      responses:
+        '200':
+          description: Pong
+        '500':
+          description: Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      operationId: ping
+      tags:
+        - test
+      security: [] # no security
+
+  '/googleproject':
+    get:
+      summary: Request a clean Google project
+      responses:
+        '200':
+          description: Project allocated successfully, here are the details
+          schema:
+            $ref: '#/definitions/Project'
+        '500':
+          description: Project could not be allocated
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      operationId: requestGoogleProject
+      tags:
+        - project
+      security:
+        - googleoauth:
+          - openid
+          - email
+          - profile
+
+  '/googleproject/{projectName}':
+      get:
+        summary: Release a Google project
+        parameters:
+          - in: path
+            name: projectName
+            description: googleProject
+            required: true
+            type: string
+        responses:
+          '200':
+            description: Project released successfully
+          '500':
+            description: Project could not be released
+            schema:
+              $ref: '#/definitions/ErrorReport'
+        operationId: releaseGoogleProject
+        tags:
+          - project
+        security:
+          - googleoauth:
+            - openid
+            - email
+            - profile
+
+##########################################################################################
+## DEFINITIONS
+##########################################################################################
+definitions:
+  Project:
+    description: ''
+    required:
+      -projectName
```

Comment:
@helgridly I get the error below pointing to the line above. Can you tell why?

![image](https://user-images.githubusercontent.com/5438223/36047681-c71612d2-0daa-11e8-8c29-e97689f07a6d.png)

Reply:
You might just need a space between the dash and projectName

Reply:
Thank you!

Reply:
As a side note, see this comment for swagger validation resources: https://broadinstitute.atlassian.net/browse/DSPTO-246?focusedCommentId=278478&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-278478

In practice, I found that you need three layers of validation to catch all potential errors:
* Intellij's swagger validation - good first pass
* Swagger's version of validation: https://editor.swagger.io/ - good second pass
* Both of those can miss some yaml formatting errors, so you need a third pass with https://codebeautify.org/yaml-validator

Reply:
Thanks @rushtong!